Client/Frontend/Browser/Onboarding/OnboardingAdsAvailableView.swift

```diff
@@ -93,13 +95,13 @@ extension OnboardingAdsAvailableController {
             
             mainStackView.addArrangedSubview(descriptionView)
 
-            [skipButton, continueButton, UIView.spacer(.horizontal, amount: 0)]
+            [skipButton, continueButton]
                 .forEach(buttonsStackView.addArrangedSubview(_:))
             
             [textStackView, buttonsStackView].forEach(descriptionStackView.addArrangedSubview(_:))
             
-            continueButton.snp.makeConstraints {
-                $0.centerX.equalTo(self.snp.centerX)
+            skipButton.snp.makeConstraints {
```

Comment:
Remove this and use `fillEqually` distribution of the button's stack view. Should work the same

Reply:
There's no `fillEqually` in my PR :S

Reply:
Ahh I see what happened. The comment is older than the PR. I had already removed it before seeing this comment.

Reply:
It's better to use `fillEqually` distribution than to use constrains inside of stack view imo